### ISSUE TITLE
Mara mobs outside dungeon should be elite

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#6202]
> Mara mobs outside dungeon should be elite

.go xyz -1239.450000 2861.880000 119.627000 1 3.731430

**Comment by zana244** [comment]
TBH I also remember some mobs being elite... most of them are normal, it seems like Maraudine Scouts might be elite on classic?
https://www.youtube.com/watch?v=XCCQbEtMMjE
However wowhead lists them all as normal...

**Comment by Shadownflame** [comment]
Yep the scouts (NPC ID 4654) were elites for some reason, but they had normal mob health, just higher damage. The other mobs have never been elites.
